google/resource_google_organization_policy.go

```diff
@@ -238,6 +273,23 @@ func expandBooleanOrganizationPolicy(configured []interface{}) *cloudresourceman
 	}
 }
 
+func expandRestoreOrganizationPolicy(configured []interface{}) (*cloudresourcemanager.RestoreDefault, error) {
+	if len(configured) == 0 {
+		return nil, nil
+	}
+
+	if len(configured) > 0 {
```

Comment:
This if statement isn't necessary since we already checked above if it's equal to 0

Reply:
because there are three escenarios there:
1.- when we doesnt have the restore_policy argument.
2.-when we have the restore_policy argument, but the default value is not true.
3.-when we have the restore_policy argument and the default value is true.

Reply:
Right, it could look like this:
```
	if len(configured) == 0 {
		return nil, nil
	}

	restoreDefaultMap := configured[0].(map[string]interface{})
	default_value := restoreDefaultMap["default"].(bool)
 
	if default_value {
		return &cloudresourcemanager.RestoreDefault{}, nil
	}

	return nil, fmt.Errorf("Invalid value for restore_policy. Expecting default = true")
```

Reply:
yeah, one 'if' is not necessary and looks cleaner, good catch (changed).